stdlib/cmake/modules/AddSwiftStdlib.cmake

```diff
@@ -297,12 +301,6 @@ function(_add_target_variant_c_compile_flags)
   endif()
 
   if("${CFLAGS_SDK}" STREQUAL "ANDROID")
-    list(APPEND result -nostdinc++)
```

Comment:
Without this flag, clang automatically finds the libc++ headers in the unified sysroot and `swift_android_libcxx_include_paths` is no longer needed.